iphone/Classes/TiUIiOSProxy.m

```diff
@@ -841,5 +841,16 @@ - (id)createWebViewProcessPool:(id)args
 MAKE_SYSTEM_PROP(INJECTION_TIME_DOCUMENT_END, WKUserScriptInjectionTimeAtDocumentEnd);
 #endif
 
+- (TiColor *)fetchSemanticColor:(id)color
+{
+  ENSURE_SINGLE_ARG(color, NSString);
+
+  if ([TiUtils isIOSVersionOrGreater:@"11.0"]) {
```

Comment:
I think we'll want to merge this after https://github.com/appcelerator/titanium_mobile/pull/11091 so I can wrap this in a check?